cmake/external/protobuf.cmake

```diff
@@ -236,20 +236,20 @@ function(build_protobuf TARGET_NAME BUILD_FOR_HOST)
 
   if(WITH_ASCEND AND NOT WITH_ASCEND_CXX11)
     set(PROTOBUF_REPOSITORY https://gitee.com/tianjianhe/protobuf.git)
-    set(PROTOBUF_TAG v3.8.0)
+    set(PROTOBUF_TAG v3.19.0)
   elseif(WITH_ASCEND_CL AND NOT WITH_ASCEND_CXX11)
     set(PROTOBUF_REPOSITORY https://gitee.com/tianjianhe/protobuf.git)
-    set(PROTOBUF_TAG v3.8.0)
+    set(PROTOBUF_TAG v3.19.0)
   elseif(WITH_IPU)
     set(PROTOBUF_REPOSITORY ${GIT_URL}/protocolbuffers/protobuf.git)
-    set(PROTOBUF_TAG d750fbf648256c7c631f51ffdbf67d7c18b0114e)
+    set(PROTOBUF_TAG v3.19.0)
```

Comment:
hi, 对于ipu来说, 这里的版本需要保持原来的tag, 不然会遇到 符号冲突问题(与ipu的库使用的 版本不一致), 需要更多的时间来解决/验证

类似于这个
https://github.com/protocolbuffers/protobuf/issues/3899

Reply:
已经跟 @qili93 讨论过升级protobuf对ipu的影响，也知道ipu希望保持原tag。这个PR只是测试用，如果ipu无法支持就不会修改这一块的tag。
另外再确认下，用户环境里安装protobuf 4.x版本，ipu可以正常使用paddle吗？

Reply:
> 用户环境里安装protobuf 4.x版本，ipu可以正常使用paddle吗？

不能, 会报错

```
>>> import paddle
Warning with import paddle: you should not
     import paddle from the source directory; please install paddlepaddle*.whl firstly.Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/alleng/repos/Paddle_internal/python/paddle/__init__.py", line 25, in <module>
    from .framework import monkey_patch_variable
  File "/alleng/repos/Paddle_internal/python/paddle/framework/__init__.py", line 17, in <module>
    from . import random  # noqa: F401
  File "/alleng/repos/Paddle_internal/python/paddle/framework/random.py", line 16, in <module>
    import paddle.fluid as fluid
  File "/alleng/repos/Paddle_internal/python/paddle/fluid/__init__.py", line 36, in <module>
    from . import framework
  File "/alleng/repos/Paddle_internal/python/paddle/fluid/framework.py", line 35, in <module>
    from .proto import framework_pb2
  File "/alleng/repos/Paddle_internal/python/paddle/fluid/proto/framework_pb2.py", line 36, in <module>
    type=None),
  File "/opt/conda/lib/python3.7/site-packages/google/protobuf/descriptor.py", line 755, in __new__
    _message.Message._CheckCalledFromGeneratedFile()
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).

More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates
```